tests/models/test_modeling_common.py

```diff
@@ -565,3 +576,65 @@ def test_deprecated_kwargs(self):
                 f" {self.model_class}.__init__ if there are deprecated arguments or remove the deprecated argument"
                 " from `_deprecated_kwargs = [<deprecated_argument>]`"
             )
+
+
+@is_staging_test
+class ModelPushToHubTester(unittest.TestCase):
+    identifier = uuid.uuid4()
+    repo_id = f"test-model-{identifier}"
```

Comment:
```suggestion
    repo_id = f"test-model-{identifier}"
    org_repo_id = f"valid_org/{repo_id}-org"
```

(nit) I would define the `org_repo_id` at the same place as the `repo_id` for consistency and avoid re-computing it 5 times in `test_push_to_hub_in_organization`

Reply:
I will settle with whatever currently is for now :)